### PR TITLE
feature:review

### PR DESCRIPTION
**Pull request by RianIhsan**
image

### CODE REVIEW

File: module/product/domain/models.go [hunk 1]
```diff
@@ -1,6 +1,10 @@
...
 
-import "time"
+import (
+	"time"
+
+	"github.com/capstone-kelompok-7/backend-disappear/module/review/domain"
+)
 
 type Product struct {
 	ID            int             `gorm:"column:id;type:int;primaryKey" json:"id"`
```
Comment (masnann) [inline module/product/domain/models.go:10]: ganti jadi product models, bikin function buat table name

File: module/review/domain/models.go [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package domain
+
+import "time"
+
+type Review struct {
```
Comment (masnann) [inline module/review/domain/models.go:5]: ganti seperti produk

**Review by masnann — CHANGES_REQUESTED** [review]

File: module/product/domain/models.go (hunk 1 above)
Comment (RianIhsan) [reply module/product/domain/models.go]: done

File: module/review/domain/models.go (hunk 2 above)
Comment (RianIhsan) [reply module/review/domain/models.go]: done

**Review by masnann — APPROVED** [review]

**Pull request merged**